db/migrate/20160214115800_inline_ems_id.rb

```diff
@@ -28,7 +28,7 @@ def add_columns
 
   def update_columns
     say_with_time("Inline ems_id in containers") do
-      Container.includes(:container_definitions => :container_group).all.each do |container|
+      Container.includes(:container_definition => :container_group).all.each do |container|
```

Comment:
Yeah when the association is correct it is fetched and when it isn't it will degrade to `SELECT  "containers".* FROM "containers" `. 

That happens for app/models too, e.g `Vm.includes(:hosts)`

Thanks for the fix